### DISCUSSION TITLE
BerkleyDB and GetText

### DISCUSSION THREAD

**Discussion by pde-cds** [AmpScm/SharpSvn#16] [Q&A]
I've cloned and build the repo, in the licenses there is the BerkleyDB license  and GetText License which are a both a copyleft.
Does the nuget package include the BerkleyDB and GetText-Runtime because this might prevent devs from using it.

**Comment by markusschaber** [comment]
As fare as I know, there was some option to build without BDB support (which is neded for file:// access to repositories in the old bdb format), it might even be the default.
I think the GetText is only used by some command line tools (sliksvn?), and not the SharpSVN library.
Maybe you could just investigate what's inside the NuGet package?

**Reply by pde-cds** [reply]
Inside the nuget package there is only the SharpSvn.dll, a corresponding .xml, a SharpPlink-Win32.svnExe and SharpSvn-DB44-20-win32.svnDll

**Reply by pde-cds** [reply]
For clearification i just wanted to analyse the project for unwanted licenses but want to use the prebuild nuget package from nuget.org.
Therefore it would suffice to know, if the BerkeleyDB and GetText licenses are part of those binaries, or if they're only needed to build it. I suppose BerkeleyDB is only part of the Subversion server, and GetTexts seems to be part of the build toolchain.
